numbers/guides/csrLookup.md

```diff
@@ -0,0 +1,384 @@
+| `<BasicAuthentication>`  | -                          | Basic auth credentials to apply to your message & voice events                                                                                                                                                                                                                                                                                                                                                  |
+| `Username`               | No, but highly recommended | Basic auth `Username`                                                                                                                                                                                                                                                                                                                                                                                           |
+| `Password`               | No, but highly recommended | Basic auth `Password`                                                                                                                                                                                                                                                                                                                                                                                           |
+| `<PublicKey>`            | No                         | BASE64 encoded public key matching the notification receiving server                                                                                                                                                                                                                                                                                                                                            |
+
+{% common %}
+
+### Create Subscription
+
+{% sample lang="http" %}
+
+```http
+POST https://dashboard.bandwidth.com/api/accounts/{{accountId}}/subscriptions HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+
+<Subscription>
+  <OrderType>csrs</OrderType>
+  <CallbackSubscription>
+    <URL>{your-callback-url}</URL>
+    <Expiry>3153600000</Expiry>
+    <CallbackCredentials>
+      <BasicAuthentication>
+          <Username>User15</Username>
+          <Password>Hunter15</Password>
+      </BasicAuthentication>
+    </CallbackCredentials>
+  </CallbackSubscription>
+</Subscription>
+```
+
+{% common %}
+
+### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{{accountId}}/subscriptions/{{applicationID}}
+```
+
+{% endextendmethod %}
+
```

Comment:
Do we need this in here? We already have create subscriptions documented

Reply:
Yes, as it's part of the holistic setup XP. Sadly still no UI for subscriptions. I updated with a disclaimer that you probably only need to do once.

Good call out!